pai-experimental/cdp.sol

```diff
@@ -314,8 +316,12 @@ contract CDP is DSMath, DSNote, Template {
 
     /// liquidate a CDP
     function liquidate(uint record) public note {
-        require(!safe(record) || settlement);
+        require(!safe(record));
+        require(!settlement);
```

Comment:
we can liquidate a CDP whenever the CDP is unsafe or we are in settlement stage.

Reply:
This need to be discussed. Of course you can make the final decision.